data/mods/More_Survival_Tools/professions.json

```diff
@@ -0,0 +1,110 @@
+[
+    {
+        "type": "profession",
+        "ident": "nomad",
+        "name": "Nomad",
+        "description": "You always prefered the comfort of the open sky, far from the complexities of modern life. Though from the looks of it, things have changed since last time you've been anywhere near civilization.",
+        "points": 4,
+        "skills": [
+            {
+                "level": 4,
```

Comment:
OK.  It'd cost four points to buy this amount of survival skill at chargen: first point buys 2 and the next are one apiece.  Similar pricing applies down the line, so you're looking at 12 points just in skills.